### DISCUSSION TITLE
Community RPC Reward Distribution at Q4 of 2023

### DISCUSSION THREAD

**Discussion by fewensa** [ringdao-community-guild/discussion#60] [General]
## Target

After 2 quarters of operation, Community RPC has now almost achieved the original intention of the established program.

Reference: [Community RPC Experimental Program](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x26be609917096095e8fb2a617d6def3fa62b3240285a9887bcd9184afb2a0ddd)

## Detail about Q4 of 2023

1. This quarter, a new member has joined the crab network [Add akis crab node](https://github.com/dcdao/community-rpc/pull/25) . increased stability of crab network.
2. This quarter compared to the previous quarter (Q3 of 2023), the number of requests increased by 52% (Ref: https://community-rpc.darwiniacommunitydao.xyz/), making a strong contribution to the darwinia ecosystem.

## Reward Distribution

This quarterly distribution model will continue from the previous quarter. distribution will be based on https://community-rpc.darwiniacommunitydao.xyz indicators.
A new plan will be switched from next quarter, and there will be new document to discuss.

## Plan

Community RPC also plans to upgrade
1. A new proxy method.
2. Call statistics
3. Accurate reference data for reward distribution

**Comment by SasoLithops** [comment]
Hi!

Great work. To join one must run a node right? May I know if there are any other requierments? We can share this on socials so possibly more can join,

**Comment by fewensa** [comment]
Yes it is. when node running, post new issue https://github.com/dcdao/community-rpc/issues/new/choose your node info.

**Reply by SasoLithops** [reply]
Got it. Thank you.

**Comment by fewensa** [comment]
Sent
https://etherscan.io/tx/0xc60844aa9d9280d679ade2593d2feb86be853afdc0a2d2ac929d1844fd2b9b75
https://etherscan.io/tx/0x29c52cb10e7afe421e0d7a19625f23e79ffaf868e13411142ce278c7289e44f1
https://etherscan.io/tx/0xffce8b5551c0e6b2c855083bd2c6f8e6844f0c92d1e6ff2d6437ed0578130bc4
